.github/workflows/build-ci.yml

```diff
@@ -120,6 +120,40 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         upload_release: true
 
+  build-appimage:
```

Comment:
Not that order matters (much), but the way I see it we first build pioneer using gcc & clang, to test code compiles with both, so maybe we don't want to have the app-image build in between?

Reply:
done...